### ISSUE TITLE
Alternative XML Syntax

### ISSUE THREAD

**Issue by h3rald** [h3rald/glyph#110]
Implement the possibility to "fallback" to create an HTML (or XML for all that matters) tag if a macro is not found.

e.g.:
##       p[This is a paragraph]

```
  <p>This is a paragraph</p>
```

or even:
##       span[@class[test]contents...]

```
  <span class="test">contents...</span>
```

To force XML fallback, it should be possible to prepend a "=" to any macro, e.g.

```
=code[puts "hello"]
```

This should be governed by configuration settings, e.g.
    language.set = 'glyph' # default
    # 'xml', 'bare' 
    language.options.xml_fallback = true

**Comment by h3rald** [comment]
Working on this.

**Comment by h3rald** [comment]
Removed div macro.

**Comment by h3rald** [comment]
Closed by 51703e0332255945aa09.
